### DISCUSSION TITLE
How to add heading or paragraph to list items ?

### DISCUSSION THREAD

**Discussion by WoodenPC** [facebook/lexical#5770] [Q&A] [answered]
Hi, can i wrap list item with heading or paragraph ? For example this is how it works in ckeditor https://ckeditor.com/ckeditor-5/demo/feature-rich/
![image](https://github.com/facebook/lexical/assets/25397745/e73a5e8e-6fb9-4818-910e-35812b3a281c)
In lexical playground, i can only replace heading to list, or heading to paragraph etc...

**Comment by ivailop7** [comment, marked as answer]
There's an open issue, if you'd like to fix: https://github.com/facebook/lexical/issues/5646
